.travis.yml

```diff
@@ -60,9 +59,34 @@ jobs:
     # Make sure the instances listed below match up with
     # the `platforms` defined in `kitchen.yml`
     - env: INSTANCE=java-debian-10-master-py3
-    - env: INSTANCE=cpp-centos-8-master-py3
-    - env: INSTANCE=java-fedora-31-master-py3
-    - env: INSTANCE=javascript-opensuse-leap-151-master-py3
+    # - env: INSTANCE=default-ubuntu-1804-master-py3
+    - env: INSTANCE=java-centos-8-master-py3
+    - env: INSTANCE=javascript-fedora-31-master-py3
+    - env: INSTANCE=cpp-opensuse-leap-151-master-py3
+    # - env: INSTANCE=default-amazonlinux-2-master-py3
+    # - env: INSTANCE=default-debian-10-3000-2-py3
+    # - env: INSTANCE=default-debian-9-3000-2-py3
+    # - env: INSTANCE=default-ubuntu-1804-3000-2-py3
+    # - env: INSTANCE=default-centos-8-3000-2-py3
+    # - env: INSTANCE=default-centos-7-3000-2-py3
+    # - env: INSTANCE=default-fedora-31-3000-2-py3
+    # - env: INSTANCE=default-opensuse-leap-151-3000-2-py3
+    # - env: INSTANCE=default-amazonlinux-2-3000-2-py3
+    # - env: INSTANCE=default-ubuntu-1804-3000-2-py2
+    # - env: INSTANCE=default-ubuntu-1604-3000-2-py2
+    # - env: INSTANCE=arch-arch-base-latest-3000-2-py2
+    # - env: INSTANCE=default-debian-10-2019-2-py3
+    # - env: INSTANCE=default-debian-9-2019-2-py3
+    # - env: INSTANCE=default-ubuntu-1804-2019-2-py3
+    # - env: INSTANCE=default-ubuntu-1604-2019-2-py3
+    # - env: INSTANCE=default-centos-8-2019-2-py3
+    # - env: INSTANCE=default-centos-7-2019-2-py3
+    # - env: INSTANCE=default-fedora-31-2019-2-py3
+    # - env: INSTANCE=default-opensuse-leap-151-2019-2-py3
+    # - env: INSTANCE=default-amazonlinux-2-2019-2-py3
+    # - env: INSTANCE=default-centos-6-2019-2-py2
+    # - env: INSTANCE=default-amazonlinux-1-2019-2-py2
```

Comment:
Since there's no `default` suite (nor `arch`), these lines aren't helpful since if they're uncommented, they'll simply fail (i.e. `kitchen list`).  I'll try to remove them in a subsequent commit.  To keep placeholders here, use one of the relevant suites in place of `default`.